lambda/java-project/src/main/java/app/babelfish/LambdaHandler.java

```diff
@@ -68,7 +68,7 @@ private String saveOnS3(String bucket, String outputFile) {
 		String fileName = "output/" + new Date().getTime() + ".mp3";
 		
 		PutObjectRequest request = new PutObjectRequest(bucket, fileName, new File(outputFile));
-		request.setCannedAcl(CannedAccessControlList.PublicRead);
+		# request.setCannedAcl(CannedAccessControlList.PublicRead); // We dont want to set Public Read as we will access via OAI
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;Invalid Character&#34;, delete this token